src/Integration.Vsix.UnitTests/CFamily/CFamilyHelperTests.cs

```diff
@@ -39,11 +38,35 @@ namespace SonarLint.VisualStudio.Integration.Vsix.CFamily.UnitTests
     [TestClass]
     public class CFamilyHelperTests
     {
+        [TestMethod]
+        public void CreateRequest_IRequestPropertiesAreSet()
+        {
+            var analyzerOptions = new CFamilyAnalyzerOptions();
+            var rulesConfig = Mock.Of<ICFamilyRulesConfig>();
+
+            var request = (IRequest)GetSuccessfulRequest(analyzerOptions, "d:\\xxx\\fileToAnalyze.txt", rulesConfig);
+            request.Should().NotBeNull();
+
+            request.File.Should().Be("d:\\xxx\\fileToAnalyze.txt");
+            request.PchFile.Should().Be(SubProcessFilePaths.PchFilePath);
+            request.AnalyzerOptions.Should().BeSameAs(analyzerOptions);
+            request.RulesConfiguration.Should().BeSameAs(rulesConfig);
+        }
+
+        [TestMethod]
+        public void CreateRequest_FileConfigIsSet()
+        {
+            var request = GetSuccessfulRequest();
+            request.Should().NotBeNull();
+
+            request.FileConfig.Should().NotBeNull();
+        }
+
         [TestMethod]
         public void CreateRequest_HeaderFile_IsSupported()
         {
             // Arrange
-            var loggerMock = new Mock<ILogger>();
+            var logger = Mock.Of<ILogger>();
```

Comment:
Miscellaneous cleanup. `Mock.Of<>` makes it clearer that the object isn't important for the test.